input/papers/2016/2016arXiv160908671H/tev-000095.yaml

```diff
@@ -0,0 +1,26 @@
+paper_id: 2016arXiv160908671H
+source_id: 95
+
+data:
+  livetime: 13.4 # year 2012
+#  livetime: 65.3 year 2011, earlier observations have already been published
+#  significane: 3, 5, 7, 9 mentioned in paper fig 4
```

Comment:
Remove the commented-out line about significance.
This source is so bright and the exposure so long, that the detection significance is very high and not even mentioned in the paper, i.e. that info isn't available.